packages/vue/src/pages/planning-task-list/components/task-list-kanban/components/task-kanban-card.vue

```diff
@@ -10,7 +10,7 @@
       <div class="card-priority" />
       <dx-button
         icon="edit"
-        @click="notifyByCard()"
+        @click="notifyByCard"
```

Comment:
Use .stop modifier instead of vent.event.stopPropagation  https://v2.vuejs.org/v2/guide/events.html#Event-Modifiers